projects/crossplane/governance-review/2025-10-06.md

```diff
@@ -0,0 +1,503 @@
+<!--
+Template: https://raw.githubusercontent.com/cncf/toc/refs/heads/main/toc_subprojects/project-reviews-subproject/governance-review-template.md
+Issue: https://github.com/cncf/toc/issues/1887
+Submission: https://github.com/cncf/toc/issues/1788
+-->
+
+# Governance Review for Crossplane Graduation
+
+What follows is a governance review for the [Crossplane](https://crossplane.io)
+project. The review was conducted as part of due diligence for Crossplane's
+application to move to CNCF's Graduated level in
+<https://github.com/cncf/toc/issues/1788>.
+
+**The review primarily consists of an audit on the project's
+self-assessment in their Graduation application.**
+
+- Graduation application submission: <https://github.com/cncf/toc/issues/1788>
+- Previous submissions (for information only): <https://github.com/cncf/toc/issues/1397>, <https://github.com/cncf/toc/pull/1254>
+
+## Summary and Assessment
+
+**Status:** Satisfactory
+
+### Executing the Assessment
+
+Mario Fahlandt (@mfahlandt) and Josh Gavant (@joshgav) conducted this review in
+September-October 2025. The review follows CNCF TOC's [governance review
+template](https://github.com/cncf/toc/blob/main/toc_subprojects/project-reviews-subproject/governance-review-template.md)
+and [accompanying
+guidelines](https://github.com/cncf/toc/tree/main/toc_subprojects/project-reviews-subproject).
+
+### Must-Fix Items
+
+**The following issues have been identified that need to be resolved before
+graduation:**
+
+- The list of project maintainers in
```

Comment:
just for visibility, we are expecting this to be resolved with open PR:

* https://github.com/cncf/foundation/pull/1143